README.rst

```diff
@@ -0,0 +1,151 @@
+Nameko eventlog dispatcher
+==========================
+
+.. pull-quote::
+
+    `Nameko <http://nameko.readthedocs.org>`_ dependency provider that
+    dispatches log data using ``Events`` (Pub-Sub).
+
+
+.. image:: https://travis-ci.org/sohonetlabs/nameko-eventlog-dispatcher.svg?branch=master
+
+
+Usage
+-----
+
+Dispatching event log data
+``````````````````````````
+
+Include the ``EventLogDispatcher`` dependency in your service class:
+
+.. code-block:: python
+
+    from nameko.rpc import rpc
+    from nameko_eventlog_dispatcher import EventLogDispatcher
+
+
+    class FooService:
```

Comment:
Maybe instead of `FooService` and `BarService` in the examples, we could use more meaningful names? Like `PublisherService` and `ListenerService` or something like that? I know the internet is made of foo/bar, but I really dislike it. If you don't feel like it though there is no need to change it.